client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+const _ = require("lodash");
```

Comment:
Let's keep the polyfill for now. I believe IE10 should be still supported. I will take a look at browser support some time later.